ansible/roles/jenkins-worker/tasks/partials/tap2junit/ibmi.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+#
+# install tap2junit ibmi
+#
+
+- name: install tap2junit
+  pip: name=tap2junit state=present
+  executable: pip2
```

Comment:
TASK [jenkins-worker : prepare installing tap2junit] ******************************************
fatal: [test-iinthecloud-ibmi72-ppc64_be-1]: FAILED! => {"reason": "'executable' is not a valid attribute for a Task\n\nThe error appears to have been in '/home/sam/w/core/build/ansible/roles
/jenkins-worker/tasks/partials/tap2junit/ibmi.yml': line 7, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name
: install tap2junit\n  ^ here\n\nThis error can be suppressed as a warning using the \"invalid_task_attribute_failed\" configuration"}


thoughts?